### ISSUE TITLE
[Bug?]: Not sure if i can define it as a bug but its horribly annoying. All tamed chocobo teleport with me.

### ISSUE THREAD

**Issue by SCBakker** [Mrbysco/ChocoCraft4#76]
### General Info

- [X] I am running a modpack
- [ ] I can reproduce this issue consistently in single-player
- [X] I can reproduce this issue consistently in multi-player
- [X] I have searched for this issue previously and it was either (1) not previously reported, or (2) previously fixed and I am having the same problem.
- [ ] I am crashing and can provide my crash report(s)
- [ ] I am using the latest version of the mod

### Loader version

Craft to exile 2-0.5.2b  (Curseforge)

### Minecraft version

1.20.1

### Mod version

05.2.b

### Java version

Java SE 17.0.10

### Issue Description

Running this mod in the Craft of Exile 2 pack.

Every tamed chocobo teleports when i use a sharestone (Modpack Waystones) When i use a waystone nothing seems to happen and they all stay put.

When i use any other form of teleporting they seem to stay put. (this is a problem for me since i get around the game with sharestones)

**Comment by Mrbysco** [comment]
It's probably the waystone mod teleporting your nearby pets with you

**Comment by SCBakker** [comment]
Waystone config says it aint,

**Comment by SCBakker** [comment]
Its made to only take leashed pets with you (this work with every other mob)

**Comment by Mrbysco** [comment]
Yeah I have no clue, there's no code in Chococraft that would cause this behavior
